packages/eds-core-react/src/components/next/Introduction.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+
+<Meta title="Next (EDS 2.0)/Introduction" />
+
+# Next EDS 2.0 Components
+
+Welcome to the **Next EDS 2.0** component library! These components represent the future of the Equinor Design System, built with modern standards and improved flexibility.
+
+## What's Different?
+
+Next EDS 2.0 components are built with:
+
+- **Vanilla CSS** instead of styled-components
+- **EDS 2.0 tokens** with dynamic color system
+- **Modern accessibility** standards
+- **Improved performance** and smaller bundle sizes
+
+## Installation
+
+Next EDS 2.0 components are part of `@equinor/eds-core-react` and available under the `/next` subpath.
+
+### Stable Release
+
+```bash
+npm install @equinor/eds-core-react
+```
+
+Both stable and next components are included in the same package. Import from the `/next` subpath to use next components.
+
+### Beta Release (Early Access)
+
+```bash
+npm install @equinor/eds-core-react@beta
+```
+
+Beta releases allow teams to test next components in real projects and provide feedback before they graduate to stable.
+
+## Usage
+
+### Importing Components
+
+```tsx
+import { Input } from '@equinor/eds-core-react/next'
+
+function MyComponent() {
+  return <Input placeholder="Enter text" />
+}
+```
+
+### CSS Import
+
+**The CSS is automatically included** when you import components from `@equinor/eds-core-react/next`.
+
+If you need to import CSS separately (e.g., for testing or custom builds):
+
+```tsx
+import '@equinor/eds-core-react/index.css' // Development
+import '@equinor/eds-core-react/index.min.css' // Production
+```
+
+## EDS 2.0 Token System
+
+All Next components use the EDS 2.0 token system with semantic color tokens that adapt to context.
+
+### Color Appearance
+
+Components use the `data-color-appearance` attribute to control semantic context.
+
+Available appearances: `neutral`, `accent`, `success`, `info`, `warning`, `danger`
+
+### Density
+
+Control spacing density using the `data-density` attribute or `EdsProvider`.
+
+Available densities: `spacious` (default), `comfortable`.
+
+### Color Schemes
+
+Components automatically adapt to light and dark modes using `data-color-scheme` attribute.
```

Comment:
I don't think we need this documentation as part of the new components. This is handled internally. I would keep the documentation as simple as possible without implementation details.